plugins/packages/packages_renderer.rb

```diff
@@ -16,17 +16,48 @@
 # you may find current contact information at www.suse.com
 
 class PackagesRenderer < Renderer
-  def do_render
-    return unless @system_description.packages
+  def display_name
+    "Packages"
+  end
+
+  def content(description)
+    return unless description.packages
 
     list do
-      @system_description.packages.each do |p|
+      description.packages.each do |p|
         item "#{p.name}-#{p.version}-#{p.release}.#{p.arch} (#{p.vendor})"
       end
     end
   end
 
-  def display_name
-    "Packages"
+  # In the comparison case we only want to show the package name, not all details like version,
+  # architecture etc.
+  def compare_content_only_in(description)
+    list do
+      description.packages.each do |p|
+        item "#{p.name}"
+      end
+    end
+  end
+
+  def compare_content_changed(changed_elements)
```

Comment:
Assignment Branch Condition size for compare_content_changed is too high. [20.32/15]